packages/types/src/discord/guild.ts

```diff
@@ -646,6 +667,7 @@ export interface DiscordCreateGuildChannel {
   default_sort_order?: SortOrderTypes | null
 }
 
+// TODO: We usually don't have types for the JSON request params, we should probably remove this
```

Comment:
We already have (camelized) types for these, I think we should remove these since they're not used anywhere